src/pages/home/home.ts

```diff
@@ -46,6 +52,33 @@ export class HomePage {
       .catch(error => alert(`An error has occured:\n ${error}`));
   }
 
+  presentProfileModal() {
+    const profileData = this.getProfileData("email", "firstname", "lastname");
+    console.log("firstname " + profileData.firstname);
+    const profileModal = this.modalCtrl.create(ProfilePage, {profileData});
```

Comment:
I'm not sure how this is going to look but the profile page should not be a modal, it should just be it's own page

Reply:
Addressed all comments but this one, when trying to use a normal page there is a weird lag when switching pages. I will look into it further tomorrow.